server/src/main/java/org/jboss/as/server/DomainServerMain.java

```diff
@@ -81,6 +81,15 @@ public static void main(String[] args) {
         } catch (Throwable ignored) {
         }
 
+        // This message is not used to display any information on the standard output, it signals the Process Controller
+        // to switch the standard output of this managed process from the process controller log to the process controller
+        // standard output.
+        // Once the StdioContext gets installed, both the stout and stderr of this managed process will be captured,
+        // aggregated and handled by this process logging framework.
+        // Signalling the Process Controller to switch also the System.err is optional since after installing StdioContext,
+        // there won't be more logs sent to the standard Err.
```

Comment:
thinking a bit more, although optional, the aggregation of the StdError and Stout and where they are redirected will be handled by the `logging.configuration` file, so assuming that both will be redirected to the Stout is weak, since I guess that can be changed via the logging configuration file and instead of using `handler.CONSOLE.target=SYSTEM_OUT` use `handler.CONSOLE.target=SYSTEM_ERR`, so probably this should be changed.

Reply:
Done